src/UI/styles/autofill-tooltip-styles.css

```diff
@@ -14,8 +14,6 @@
     BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu',
     'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif;
     -webkit-font-smoothing: antialiased;
-    /* move it offscreen to avoid flashing */
-    transform: translate(-1000px);
```

Comment:
I don't think this was needed any longer because now the tooltip is hidden before it's positioned so we should be good. I tried testing slowing down the CPU as well, so I think we're fine, but let me know if you notice anything weird.